src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -2264,15 +2266,20 @@ def create_lb_frontend_ip_configuration(
 
 def set_lb_frontend_ip_configuration(
         cmd, instance, parent, item_name, private_ip_address=None,
-        private_ip_address_allocation=None, public_ip_address=None, subnet=None,
-        virtual_network_name=None, public_ip_prefix=None):
+        private_ip_address_allocation=None, public_ip_address=None,
+        subnet=None, virtual_network_name=None, public_ip_prefix=None):
     PublicIPAddress, Subnet, SubResource = cmd.get_models('PublicIPAddress', 'Subnet', 'SubResource')
-    if private_ip_address == '':
+    if not private_ip_address:
         instance.private_ip_allocation_method = 'dynamic'
         instance.private_ip_address = None
+        # Doesn't support update operation for now
```

Comment:
small but this code could be just moved outside as it's duplicated?

Reply:
done